cookbooks/apt/recipes/default.rb

```diff
@@ -158,7 +158,7 @@
   uri "https://releases.wikimedia.org/debian"
   distribution "jessie-mediawiki"
   components ["main"]
-  key "90E9F83F22250DD7"
+  key "AF380A3036A03444"
```

Comment:
If the key has changed, which I doubt as we would be getting errors, then this should be fixed in a separate pull really.

Reply:
According to https://www.mediawiki.org/wiki/Parsoid/Setup#Ubuntu_/_Debian, "AF380A3036A03444" is the correct key. "90E9F83F22250DD7" did not work for me when testing: Parsoid came up as an unavailable package.

Reply:
You're quite right - they updated it recently it seems. I though that `apt-get update` exited non-zero when that happens, which makes it quite noticeable, but apparently it just skips updating that repo :-(

I've pushing a fix for that now.